resources/qml/Menus/ConfigurationMenu/ConfigurationItem.qml

```diff
@@ -7,143 +7,111 @@ import QtQuick.Controls 2.0
 import UM 1.2 as UM
 import Cura 1.0 as Cura
 
-Rectangle
+Button
 {
     id: configurationItem
 
     property var configuration: null
-    property var selected: false
-    signal activateConfiguration()
 
     height: childrenRect.height
-    border.width: UM.Theme.getSize("default_lining").width
-    border.color: updateBorderColor()
-    color: selected ? UM.Theme.getColor("configuration_item_active") : UM.Theme.getColor("configuration_item")
-    property var textColor: selected ? UM.Theme.getColor("configuration_item_text_active") : UM.Theme.getColor("configuration_item_text")
 
-    function updateBorderColor()
+    background: Rectangle
     {
-        border.color = selected ? UM.Theme.getColor("configuration_item_border_active") : UM.Theme.getColor("configuration_item_border")
-    }
-
-    Column
-    {
-        id: contentColumn
-        width: parent.width
-        padding: UM.Theme.getSize("default_margin").width
-        spacing: Math.round(UM.Theme.getSize("default_margin").height / 2)
+        height: childrenRect.height
```

Comment:
Background item scales by itself. It shouldn't be needed to set its height.

Reply:
It is necessary, because the background item scales to its parent's height instead of its children's height. We want it to scale to its children's height here.